### PR TITLE
Fix 1805 Default collapsed Menu in Admin Section 

### LINKED ISSUE #1805: Default collapsed Menu in Admin Section
**I'm submitting a ...**
- [x] feature request

**Current behaviour:**

screenshot from 2018-10-15 12-35-11

**Expected behaviour:**

Should be collapsed.

Will work on issue??

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1805 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

screenshot from 2018-10-16 14-20-58
screenshot from 2018-10-16 14-21-02
screenshot from 2018-10-16 14-21-04
screenshot from 2018-10-16 14-21-06
screenshot from 2018-10-16 14-21-11

### CODE REVIEW

**Comment by sk9331657** [comment]
@gabru-md @palnabarun @ParthS007  @Orbiter  Please review this.

**Comment by sk9331657** [comment]
@yashLadha

File: frontend/app/index.html [hunk 1]
```diff
@@ -16,7 +16,10 @@
     <meta name="twitter:url" content="https://twitter.com/fossasia">
     <meta name="twitter:title" content="Badgeyay - FOSSASIA">
     <meta name="twitter:description" content="Badgeyay is a simple badge generator with a simple web UI.">
-
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
```
Comment (ParthS007) [inline frontend/app/index.html]: Please follow the frameworks and UI library we are using for the project.

File: frontend/app/index.html [hunk 2]
```diff
@@ -16,7 +16,10 @@
...
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```
Comment (ParthS007) [inline frontend/app/index.html]: Same

File: frontend/app/templates/admin.hbs [hunk 3]
```diff
@@ -1,36 +1,44 @@
-<div class="row">
-  <div class="x-scrollable sixteen wide column">
-    {{#tabbed-navigation}}
-      {{#link-to 'admin.index' class='item'}}
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```
Comment (ParthS007) [inline frontend/app/templates/admin.hbs]: We are using Semantic UI for styling UI.

**Comment by ParthS007** [comment]
@sk9331657 Please follow the frameworks and UI library we are using for the project. I am closing this PR.

**Comment by sk9331657** [comment]
@ParthS007  Seems Like Semantic UI doesn't have reactive nav bar that's why I used bootstrap

**Pull request closed without merging**